crates/re_types_builder/src/codegen/python.rs

```diff
@@ -1049,8 +1054,11 @@ fn quote_field_converter_from_field(
             // we generate a default converter only if the field's type can be constructed with a
             // single argument
             if field_obj.fields.len() == 1 || field_obj.is_union() {
-                let converter_name =
-                    format!("_{}_{}_converter", obj.name.to_lowercase(), field.name);
+                let converter_name = format!(
+                    "_{}__{}__special_field_converter_override", // TODO(emilk): why does this have an underscore prefix?
```

Comment:
> // TODO(emilk): why does this have an underscore prefix?

Pretty sure that's just the convention to express private-ness in python isn't it?